.github/workflows/codespell.yml

```diff
@@ -0,0 +1,19 @@
+name: Codespell
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  spell-check:
+    if: github.event.pull_request.draft == false
```

Comment:
❤️ 
That's cool, although does mean someone couldn't find typos in their PR until it's open for review...